metricbeat/module/http/json/json_integration_test.go

```diff
@@ -15,32 +15,53 @@ import (
 func TestFetch(t *testing.T) {
 	compose.EnsureUp(t, "http")
 
-	f := mbtest.NewEventFetcher(t, getConfig())
+	f := mbtest.NewEventsFetcher(t, getConfig("object"))
 	event, err := f.Fetch()
 	if !assert.NoError(t, err) {
 		t.FailNow()
 	}
 
 	t.Logf("%s/%s event: %+v", f.Module().Name(), f.Name(), event)
+
+	f = mbtest.NewEventsFetcher(t, getConfig("array"))
```

Comment:
Can we split this up in 2 tests? Like TestFetchObject and TestFetchArray? This will make it eaiser to spot what failed if something fails